bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Cursor.java

```diff
@@ -319,6 +338,55 @@ public Cursor(Device device, ImageData source, int hotspotX, int hotspotY) {
 	init();
 }
 
+/**
+ * <b>IMPORTANT:</b> This method is not part of the public
+ * API for Image. It is marked public only so that it
+ * can be shared within the packages provided by SWT. It is not
+ * available on all platforms, and should never be called from
+ * application code.
+ *
+ * Updates zoom and refresh the Cursor based on the native zoom level, if required.
+ *
+ * @param cursor the cursor to get the handle of
+ * @param zoom device zoom in % of the monitor on which the image is painted
+ *
+ * @return true if cursor is refreshed
+ *
+ * @noreference This method is not intended to be referenced by clients.
+ */
+public static Long win32_getHandle (Cursor cursor, int zoom) {
+	if (cursor.isDisposed()) {
+		return cursor.handle;
```

Comment:
Why when it's disposed? Shouldn't it be:
```suggestion
	if (!cursor.isDisposed()) {
		return cursor.handle;
```

?

Reply:
For non-disposed case we have the handling. We add the condition to return the handle even if the resource is disposed. Same as its done with Image.